spec/integration/create_subscriber_list_spec.rb

```diff
@@ -80,11 +80,11 @@
       expect(response.status).to eq(422)
     end
 
-    it "returns an error if creating the same topic" do
+    it "successfully creates two SubscriberList objects with the same title" do
       create_subscriber_list(title: "oil and gas", tags: { topics: ["oil-and-gas/licensing"] })
       create_subscriber_list(title: "oil and gas", tags: { topics: ["oil-and-gas/licensing"] })
```

Comment:
I'd perhaps change the test here to reflect a more reasonable scenario, e.g. having the 2nd subscriber list be a policy area subscription for example?

Reply:
Agreed 👍